### PR TITLE
#146 #239 fix first 5 steps

### PR DESCRIPTION
**Pull request by agamaloni**

### CODE REVIEW

**Comment by danielhers** [comment]
Looks good. There's just a missing י in ז'בוטינסק in one place. 
Does the "previous" button show in the first step? It shouldn't.

**Comment by agamaloni** [comment]
add simple changes
please see that index.html changed only for id="step3tour" and id="step4tour"

**Comment by danielhers** [comment]
You seem to have reverted some changes I did in recent commits in `templates/index.html`, e.g. "Anyway" -> "ANYWAY".
Please start from an up-to-date branch and apply your changes there.

File: static/js/tour.js [hunk 1]
```diff
@@ -21,53 +26,102 @@ var tour = new Tour({
...
-  {
-    element:  $('#step2Component'),
-    placement: 'left',
+        {
+    element:  '#step3tour',
     title: "סינון לפי חומרה",
-    content: '<p>כאן ניתן להציג או להסתיר תאונות ברמות חומרה שונות. </br>לצורך הדוגמא נבטל את הצגתן של תאונות קלות עך ידי ביטול הסימון.</br> במידה ותבחרו להציג תאונות שמיקומן מוערך שימו לב שמידת ההערכה תופיע בשדה "עיגון" בפרטי התאונה ואייקון התאונה יהיה שקוף. </p>'
+    placement: 'left',
+    content: '<p>כאן ניתן להציג או להסתיר תאונות ברמות חומרה שונות. </br>לצורך הדוגמא נבטל את הצגתן של תאונות קלות על ידי ביטול הסימון.</br> במידה ותבחרו להציג תאונות שמיקומן מוערך שימו לב שמידת ההערכה תופיע בשדה "עיגון" בפרטי התאונה ואייקון התאונה יהיה שקוף. </p>',
+    onShow: function(){
```
Comment (danielhers) [inline static/js/tour.js]: If the checkbox was disabled, it enables it.
To be consistent with the text it should always disable (or do nothing if already disabled).

File: static/js/tour.js [hunk 2]
```diff
@@ -21,53 +26,102 @@ var tour = new Tour({
...
+            $('*[data-type="3"]').click();
+   //     }
+        step3prev();
+        $('*[data-type="3"]').click();
+
+    }
   },
 {
-    element:  $('#step2Component'),
+    element:  '#step4tour',
```
Comment (danielhers) [inline static/js/tour.js]: "Next" button should be disabled until next step is implemented

File: static/js/tour.js (hunk 1 above)
Comment (galraij) [inline static/js/tour.js]: במקום ״עך ידי״ שיהיה ״על ידי״



‫ב-17 במאי 2015, בשעה 09:50, ‏‏Daniel Hershcovich ‏<notifications@github.com> כתב/ה:‬

Comment (danielhers) [inline static/js/tour.js]: That's what this change fixes, it changes עך to על :)

**Pull request merged**